packages/back-end/src/queryRunners/ExperimentResultsQueryRunner.ts

```diff
@@ -85,7 +85,9 @@ export function getFactMetricGroup(metric: FactMetricInterface) {
   // Ratio metrics must have the same numerator and denominator fact table to be grouped
   if (isRatioMetric(metric)) {
     if (metric.numerator.factTableId !== metric.denominator?.factTableId) {
-      return "";
+      return metric.denominator?.factTableId
+        ? `${metric.numerator.factTableId} ${metric.denominator.factTableId} (cross-table ratio metrics)`
```

Comment:
Question: If there are 2 metrics with the same 2 fact tables, but one has the value `A/B` and another `B/A`, they will get different group ids here.  Is that intentional?

Reply:
Yes, that is intentional to keep things simple, but I think we support that use case based on the latest changes I made in that PR. I'd have to confirm that numerators from the additional table work well and might imply some slightly more generic naming, which is possible. LMK if you think it's important to explore this before landing.

Reply:
I will timebox a far more generic solution.